channels/smartcard/client/smartcard_pack.c

```diff
@@ -2892,7 +2892,7 @@ LONG smartcard_pack_control_return(SMARTCARD_DEVICE* smartcard, wStream* s,
 {
 	LONG status;
 	DWORD cbDataLen = ret->cbOutBufferSize;
-	DWORD index = 0;
+	UINT32 index = 0;
```

Comment:
This should not be necessary, `UINT32` and `DWORD` are both defined to the same 32bit unsigned integer type, if not then something in winpr is missing for MinGW